lxd/storage/backend_lxd.go

```diff
@@ -1151,6 +1151,16 @@ func (b *lxdBackend) UpdateInstance(inst instance.Instance, newDesc string, newC
 	volStorageName := project.Prefix(inst.Project(), inst.Name())
 	contentType := InstanceContentType(inst)
 
+	// Check that the volume's size property isn't being changed.
+	if newConfig["size"] != "" {
+		return fmt.Errorf("Instance volume 'size' property cannot be changed")
+	}
+
+	// Check that the volume's block.filesystem property isn't being changed.
+	if newConfig["block.filesystem"] != "" {
+		return fmt.Errorf("Instance volume 'block.filesystem' property cannot be changed")
+	}
```

Comment:
That seems wrong. If the volume has a `block.filesystem` value set, it will have to be passed through `newConfig`.
The check must be that we don't allow a change of value, but having it be passed through the map needs to be allowed.

Reply:
@stgraber ah yes, I'll move further down to where the changeConfig is generated and test it from there.